### PR TITLE
Use generic socket name

### LINKED ISSUE #20: Webserver is hardcoded
We've hardcoded thin in our [nginx config](https://github.com/twingly/capistrano-twingly/blob/782c50a2aae9763c1dfd1e175aa832fd1529c6ba/lib/capistrano/twingly/tasks/nginx.rake#L17). 

We should just call it `"#{app_name}.sock"` for now. This will break all applications though, so we need a major version bump.

### PR DESCRIPTION
**Pull request by jage**
This will require a major version bump

Close #20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ set :procfile_contents, -> {
   contents << "chruby-exec #{fetch(:chruby_ruby)} -- "
   contents << "bundle exec "
   contents << "thin start "
```
Comment (walro) [inline README.md]: should `thin start` be here if it were to truly close #20?

Comment (walro) [inline README.md]: doh, the readme lolz

**Comment by walro** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (jage) [inline README.md]: I'll probably update the README with a Puma example once we get it up and running.

Comment (dentarg) [inline README.md]: ^ @jage :)

Comment (jage) [inline README.md]: Thanks for the reminder!

**Pull request merged**